templates/ntp.conf.j2

```diff
@@ -3,7 +3,15 @@
 # For more information about this file, see the man pages
 # ntp.conf(5), ntp_acc(5), ntp_auth(5), ntp_clock(5), ntp_misc(5), ntp_mon(5).
 
-driftfile /var/lib/ntp/drift
+{% if ntp_bind_only_interfaces %}
+# Controls which network addresses ntpd opens
+interface ignore wildcard
+{% for item in ntp_interfaces %}
+interface listen {{ item }}
+{% endfor %}
+{% endif %}
+
+driftfile {{ ntp_driftfile }}
```

Comment:
You added a variable for driftfile, ntp_driftfile, shouldn't a default var be added with the value replaced, /var/lib/ntp/drift,  with the var?